### ISSUE TITLE
Multi-device

### ISSUE THREAD

**Issue by elpulga** [shekohex/wadump#7]
Is there support for the multi-device beta? I can't find the WASecretBundle key on local storage.

**Comment by shekohex** [comment]
No. I've not worked on the project or used WhatsApp since a long time ago.
If you could find the implementation or something similar implemented somewhere else we could work on it in a small PRs.
